src/main/resources/lithium.mixins.json

```diff
@@ -76,6 +76,9 @@
         "world.chunk_ticking.MixinPlayerChunkWatchingManager",
         "world.chunk_ticking.MixinThreadedAnvilChunkStorage",
         "world.explosions.MixinExplosion",
+        "world.layer_caching.MixinBiomeLayerSampler",
```

Comment:
Maybe it's best to leave the mixins off by default, as we discussed on discord

Reply:
Hmm- so just remove them from this file? or would it have to involve default values in the config?